### ISSUE TITLE
Add a matrix-video-chat widget 

### ISSUE THREAD

**Issue by karlabbott** [vector-im/element-call#213]
**Describe the solution you'd like**
Users ought to be able to add a matrix-video-chat based conference to a room as a widget and have this work very similarly to the current Jitsi widget. 

**Describe alternatives you've considered**
The Jitsi widget currently fills this need, but it would be nice to have a widget that could manage a conference based on matrix-video-chat.

**Additional context**
None at this time.

**Comment by jakewb-b** [comment]
This will actually be facilitated by the plan for the first stage of Element Call integration. It's not quite going to be a widget, but it will allow the same functionality that's requested here.

I'm going to mark this as blocked, as this is effectively blocked on https://element-io.atlassian.net/browse/PSF-705 which is being done by PSF.

**Comment by robintown** [comment]
Element Web now has this

**Comment by Thatoo** [comment]
> Element Web now has this

What a bout element android and ios?
